cScripts/CavFnc/functions/modules/fn_modulePowerOutage.sqf

```diff
@@ -0,0 +1,88 @@
+#include "..\script_component.hpp";
+/*
+ * Author: Sweetwater.I
+ * This module function set damage to all lights indicated in a array within a selected radius from module position.
+ * Gives the effect of a Power Outage.
+ * 
+ * Arguments:
+ * 0: None
+ *
+ * Example:
+ * this call cScripts_fnc_modulePowerOutage
+ *
+ * Public: No
+ */
+[[], [""], []] call {
```

Comment:
what is that??? O.o

Reply:
this is enough;
```suggestion
params ["_OutageRadius", "_OutageArray", "_center"];
```

Reply:
you can also do like this:
```suggestion
params [
    ["_OutageRadius", 0, [0]], 
    ["_OutageArray", 0, [0]],
    ["_center", 0, [0]]
];
```
To add a **default** and **expected value**.
0 = number
"" = for string